docs/security/trust/trust_sandbox.md

```diff
@@ -23,12 +23,14 @@ These instructions assume you are running in Linux or Mac OS X. You can run
 this sandbox on a local machine or on a virtual machine. You will need to
 have `sudo` privileges on your local machine or in the VM.
 
-This sandbox requires you to install two Docker tools: Docker Engine and Docker
-Compose. To install the Docker Engine, choose from the [list of supported
-platforms](../../installation/index.md). To install Docker Compose, see the
+This sandbox requires you to install two Docker tools: Docker Engine >= 1.1.0
```

Comment:
Given that Docker 1.5.0 and below is no longer supported by Docker Hub, I think we should remove the version requirement here; all docs are for the current version here, wdyt?